interface/src/app/about/about.tpl.html

```diff
@@ -1,8 +1,6 @@
-<div>
-  <h1 class="page-header">
-    About ADAGE
-  </h1>
-  <p>
-    This is where some explanatory text about ADAGE should go.
-  </p>
+<div class="page-header">
+  <h1>About ADAGE</h1>
 </div>
+<p>
+  This is where some explanatory text about ADAGE should go.
+</p>
```

Comment:
@casey: I copied the texts above but commented out the second paragraph. Once the ADAGE Signature Analysis paper's link it available, we can set it and show the second paragraph.
http://165.123.67.202/#/about